codex-ui/src/styles/themes/amber.pcss

```diff
@@ -9,6 +9,10 @@
   --amber--text-secondary: #90847C;
   --amber--text: #E4D4C8;
   --amber--text-solid-foreground: #FFFFFF;
+  --amber-alert-success: #00A64D;
```

Comment:
you don't need to add the new colors, just use "solid" and specify particular theme